hacksquad_discordpy/main.py

```diff
@@ -2,12 +2,16 @@
 
 import discord
 from discord.ext import commands
-from dotenv import load_dotenv
 
 from wrappers.leaderboard import leaderboard_wrapper
 from wrappers.team import team_wrapper
 
-load_dotenv()
+try:
+    from dotenv import load_dotenv
+    load_dotenv()
+    print("Working in dev!")
+except ImportError:
+    print("Working in production!")
```

Comment:
These print are unnecessary.
If we need to print stuff, better use the logging module.